packages/firestore/src/local/target_data.ts

```diff
@@ -118,18 +118,4 @@ export class TargetData {
       this.resumeToken
     );
   }
-
-  isEqual(other: TargetData): boolean {
```

Comment:
TargetData was never passed to our custom "equals" implementation. I actually looked back at the usage and since there is only one call, I decided to inline it. So now we have no code that treats objects with isEqual and without in the same codepath.